File: w15/code_1.cpp
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner a = new Scanner(System.in);
        int num=a.nextInt();
        try {
            for(int i=0 ; i<num ; i++){
                int put=a.nextInt();
                if(put>=3)throw new PowerFailureException();
            }
        }
        catch(PowerFailureException ex) {
            System.out.println(ex.getMessage());
            //ex.printStackTrace();
        }
    }

}

class PowerFailureException extends Exception {
    public String getMessage() {
        return ("Power Failure!");
    }
}

File: w17/code_1.java
import java.util.Scanner;
public class Main {
    public static void main(String args[]){
        Scanner a = new Scanner(System.in);
        int m=a.nextInt();
        System.out.println(fib(m));
    }

    public static long fib(int n){
        if(n==1)
            return 1;
        else if (n==2)
            return 1;
        else
            return fib(n-1)+fib(n-2);
    }
}
